### ISSUE TITLE
UPX NotCompressibleException on PyInstaller 5.0

### ISSUE THREAD

**Issue by arossert** [pyinstaller/pyinstaller#6757]
Trying to upgrade to version 5.0 from 4.10 and getting this error:
```
137113 INFO: Executing - upx--lzma-qC:\users\root\AppData\Local\pyinstaller\bincache01_py38_64bit\tornado\speedups.cp38-win_amd64.pyd
The subprocess:
    upx--lzma-qC:\users\root\AppData\Local\pyinstaller\bincache01_py38_64bit\tornado\speedups.cp38-win_amd64.pyd
exited with error 1 and error output:
                           Ultimate Packer for eXecutables
                              Copyright (C) 1996 - 2020
    UPX 3.96w       Markus Oberhumer, Laszlo Molnar & John Reiser   Jan 23rd 2020

            File size         Ratio      Format      Name
       --------------------   ------   -----------   -----------
    upx: C:\users\root\AppData\Local\pyinstaller\bincache01_py38_64bit\tornado\speedups.cp38-win_amd64.pyd: NotCompressibleException

    Packed 1 file: 0 ok, 1 error.
````

* Output of `pyinstaller --version`: ```5.0```
* Version of Python: 3.8
* Platform: Windows (wine)
* How you installed Python: python.org (wine docker)
* Works on Linux and macOS

**Comment by bwoodsend** [comment]
#6735 has accidentally promoted UPX failures to errors. I didn't realise that `PyInstaller.compat.exec_command()` ignores the return code of subprocesses.

**Comment by arossert** [comment]
@bwoodsend Thanks for the quick response, is there a fix coming up soon?

**Comment by bwoodsend** [comment]
#6758

**Comment by arossert** [comment]
👍

**Comment by arossert** [comment]
@bwoodsend Thanks for merging a fix for that, when is the next version planned to be released? (as I cannot migrate to 5.0 until this fix is included)

**Comment by bwoodsend** [comment]
Whenever I get round to fixing #6759
